tests/end2end/features/urlmarks.feature

```diff
@@ -86,6 +86,17 @@ Feature: quickmarks and bookmarks
         And I run :bookmark-del http://localhost:(port)/data/numbers/5.txt
         Then the bookmark file should not contain "http://localhost:*/data/numbers/5.txt "
 
+    Scenario: Deleting the current page's bookmark if it doesn't exist
+        When I open about:blank
+        And I run :bookmark-del
+        Then the error "Bookmark 'about:blank' not found!" should be shown
+
+    Scenario: Deleting the current page's bookmark
+        When I open data/numbers/5.txt
+        And I run :bookmark-add
+        And I run :bookmark-del
+        Then the bookmark file should not contain "http://localhost:*/data/numbers/5.txt "
```

Comment:
I know it's kind of cumbersome, but please use an unused number here, which makes sure tests don't affect each other.